test/Microsoft.NET.Build.Containers.IntegrationTests/TargetsTests.cs

```diff
@@ -418,8 +418,8 @@ public void NotAllMuslRidsLogsError()
         logger.Errors.Should().ContainSingle(error => error.Message == Resources.Strings.InvalidTargetRuntimeIdentifiers);
     }
 
-    [InlineData("linux-musl-x64", "mcr.microsoft.com/dotnet/nightly/runtime-deps:8.0-alpine-aot")]
-    [InlineData("linux-x64", "mcr.microsoft.com/dotnet/nightly/runtime-deps:8.0-jammy-chiseled-aot")]
+    [InlineData("linux-musl-x64", "mcr.microsoft.com/dotnet/runtime-deps:8.0-alpine")]
+    [InlineData("linux-x64", "mcr.microsoft.com/dotnet/runtime-deps:8.0-jammy-chiseled")]
     [Theory]
     public void AOTAppsGetAOTImages(string rid, string expectedImage)
```

Comment:
Nit: let's rename this test because these images are no longer AOT images

Reply:
renamed to AOTAppsGetExpectedImages